manual/manual/tutorials/coreexamples.etex

```diff
@@ -580,6 +580,69 @@ let fixpoint f x =
 the function "f" cannot raise a "Done" exception, which removes an
 entire class of misbehaving functions.
 
+\section{Lazy expressions}
+
+OCaml allows to defer some computation, usually expensive one, until later when 
+we need that expression evaluated. The deferred expression has type "'a Lazy.t", 
+where "'a" is the type of the evaluated expression. 
+
+We use "lazy (expr)" to delay the evaluation of some expression "expr". For 
+example, we can defer the computation of "1+1" until we need the result of that
+expression, i.e. "2". 
+
+\begin{caml_example}{toplevel}
+let lazy_two = lazy (1+1);;
+\end{caml_example}
+
+Note: "'a lazy_t" is a type constructor for keyword "lazy" used by the compiler. 
+One should use type "'a Lazy.t" to work with lazy expressions. 
+
+Also note that the value of "lazy_two" is "<lazy>", which means 
+the expression is not evaluated yet and the result is unknown.
+
+When we finally need the result of the expression, we can call "Lazy.force"  
+on the lazy expression to force the evaluation of the lazy expression. "force" 
+is a function from a standard-library  
+\href{https://caml.inria.fr/pub/docs/manual-ocaml/libref/Lazy.html}
+{module "Lazy"}.
```

Comment:
(Note that this link only works in the  html version of the manual, but we can fix this point later)

Reply:
What other versions of the manual do you have in mind? 
I feel that it would be best for newcomers if we resolved this issue asap. Not working links in docs cause frustration.

Reply:
Some people read the PDF version of the manual.

As @Octachron suggested in the comment above, feel free to left this particular issue unchanged in your PR. @Octachron may be working on a convenient macro to add links to the standard library in a portable way, but for now we can leave the HTML link.

On the other hand, it would be nice if you could add more commits or rebase to address the other issues in the rest of the review.